cpp/do_module/do_module.cpp

```diff
@@ -0,0 +1,391 @@
+#include <fmt/core.h>
+#include <mgp.hpp>
+
+#include "mgclient.hpp"
+
+const char *kProcedureCase = "case";
```

Comment:
`const char *` is a global that may change.
Prefer `constexpr std::string_view` for better codegen.